hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/test/java/org/apache/hadoop/yarn/server/nodemanager/recovery/TestNMLeveldbStateStoreService.java

```diff
@@ -1756,6 +1756,12 @@ public void testStateStoreForResourceMapping() throws IOException {
     resources = rcs.getResourceMappings().getAssignedResources("numa");
     Assert.assertEquals(numaRes, resources);
     Assert.assertEquals(numaRes, resourceMappings.getAssignedResources("numa"));
+    // test removing numa resources from state store
+    stateStore.releaseNumaResource(containerId);
+    Assert.assertEquals(0, resourceMappings.getAssignedResources("numa").size());
+    // testing calling deletion of non-existing key doesn't break anything
+    stateStore.releaseNumaResource(containerId);
```

Comment:
This can be tested using try {} catch {Assert.fail}

Reply:
@PrabhuJoseph 
  thank you for reviewing the changes! 
- changed the function name 
- removed the not required tests 
- Updated the test with 2 cases 

Please review the changed patches